test/tests.js

```diff
@@ -63,10 +63,10 @@ describe( 'elasticsearch-rolling-index-name-parser', function() {
 			assert.equal( result.period, 'weekly', 'period should be weekly' );
 	
 			assert( moment.isMoment( result.startMoment ), 'start moment should be a moment' );
```

Comment:
US locale weeks start on Sunday vs ISO Monday, so most of these dates shift a day earlier.